src/applications/facility-locator/utils/helpers.jsx

```diff
@@ -78,7 +78,8 @@ export const setSearchAreaPosition = () => {
     .appendChild(searchAreaContainer);
   document
     .querySelectorAll('.mapboxgl-ctrl-top-right')
-    .forEach(el => el.remove());
+    // IE 11 doesn't support .remove() and core-js doesn't polyfill DOM methods.
```

Comment:
"I don't always leave comments in my code, but when I do it's because of Internet Explorer."

Could you maybe make this a hair clearer: `doesn't support ChildNode.remove()`?

Reply:
oh and sorry for commenting on your team's PR where I don't belong...